README.md

```diff
@@ -39,7 +39,10 @@ The workflow will be set to fail if:
 
 **Notes**
 1. Either `project-id` or `project-name` is required. An error will be thrown if neither is specified or both are specified.
-2. `base-branch-name` is required if `target-branch-name` is specified and doesn't exist yet in the project.
+2. If `project-name` is specified, and it does not exist, it will be created. The `base-branch-name` will be used as default branch, and `target-branch-name` will be used as analysis branch. 
+   If neither is specified, it will simply use the default branch.
+   If only one of the branches is specified, it will throw error during project creation.
+3. `base-branch-name` is required if `target-branch-name` is specified and doesn't exist yet in the project.
```

Comment:
These notes refer to superscripts in the action options table above (see `Required` column)

You could use [2] and [3] as references for `base-branch-name`, and just [2] for `target-branch-name`